cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -609,20 +613,55 @@ public void RemoveAntiForgeryCookie(Controller controller)
 }
 ```
 
-NB: You will need to attach the anti-forgery token to Ajax requests.
+#### Using .NET Core 2.0 or later:
+
+After .NET Core 2.0 it is possible to automatically generate and verify the antiforgery token.
 
-After .NET Core 2.0 it is possible to automatically generate and verify the antiforgery token. Forms must have the requisite helper as seen here:
+If you are using tag-helpers - which is the default for most web project templates - all forms will automatically send the anti-forgery token.
+You can check if tag-helpers are enabled by checking if your main `_ViewImports.cshtml` file contains:
+
+```csharp
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+```
+
+`IHtmlHelper.BeginForm` also sends anti-forgery-tokens automatically.
```

Comment:
Can you add a link to where that is mentioned, so that the reader is able to properly understand what's going underneath if they need to?

Reply:
I added the link where it is mentioned to the beginning of the parent section and this subsection, as it describes TagHelpers as well as the BeginForm-topic.

OK @ThunderSon ?
https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/anti-request-forgery.md#aspnet-core-antiforgery-configuration
`Similarly, IHtmlHelper.BeginForm generates antiforgery tokens by default if the form's method isn't GET.`

Reply:
Yes, makes sense. Sadly, the link to `IHtmlGelper.BeginForm` is broken in their docs